src/test/java/APITest/register_sucess.java

```diff
@@ -0,0 +1,55 @@
+package APITest;
+
+import static org.hamcrest.Matchers.*;
+
+import io.restassured.path.json.JsonPath;
+
+import static io.restassured.RestAssured.*;
+
+public class register_sucess {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+    //Register Successful
+	String response= given().spec(Utility.requestbuild()) 
+		             .body(Payload.register())
+                     .when().post("/api/register")
+                     .then().spec(Utility.responsebuild()).assertThat().body("id",equalTo(4)).extract().response().asString();
+             System.out.println(response);
+    
+     //Register Unsuccessful
+	 String unsuccessregister =given().spec(Utility.requestbuild())
```

Comment:
Each test case should be put in a separate method. Then you should call the test methods from the main method.